styles/module.css

```diff
@@ -91,8 +100,24 @@
 }
 
 @media screen and (min-width:801px) and (max-width: 1200px){
+  .mobile-icon{
+    display: none;
+  }
+  .navigation-wrapper{
+    display:block !important;
+    height:2em;
```

Comment:
Just a note:
"Using !important is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets. "
Source: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity